examples/testdata/path_matcher/envoy_config.json

```diff
@@ -248,8 +248,8 @@
                             }
                           },
                           {
+                            "extractPathParameters": true,
                             "operation": "1.examples_path_matcher_endpoints_cloudesf_testing_cloud_goog.ESPv2_Autogenerated_CORS_GetShelf",
-                            "pathParameterExtraction": {},
                             "pattern": {
                               "httpMethod": "OPTIONS",
                               "uriTemplate": "/libraries/{library_id}/shelves/{shelf_id}"
```

Comment:
This should also be `shelfId`? Can you check how the uri template is propagated to these auto-generated methods? Perhaps you do the replacement too late.

Reply:
Yes, it should be `shelfId`. 
It should be a [existing bug](https://github.com/GoogleCloudPlatform/esp-v2/blob/bd0fa867a0eefd8bec9de0db2bfe3c5253c0a448/examples/testdata/path_matcher/envoy_config.json#L259) in our code, where auto generated cors operation doesn't have snakeName to jsonName mapping.

Reply:
Created  b/170691898.